make terminology section informative

ashimura opened the pull request:
to resolve my proposed [Issue 402](https://github.com/w3c/wot-architecture/issues/402), I'd suggest we merge this pullrequest.


<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-architecture/pull/403.html" title="Last updated on Oct 21, 2019, 1:59 PM UTC (cb32100)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-architecture/403/b7a6cbc...cb32100.html" title="Last updated on Oct 21, 2019, 1:59 PM UTC (cb32100)">Diff</a>

mlagally commented:
Please use a class annotation such as:
    <section id="sec-deployment-scenario" class="informative">
        <h1>Example WoT Deployments</h1>

ashimura commented:
Ah, sorry...

The 'index.html' had explicit text saying "This section is normative.", so I simply changed the "normative" part to "non-normative". However, yes, it would be better to use "informative" class notation.

So I've just removed the block saying "This section is normative." and added the "informative" class attribute to the "<section>" tag instead :)

mmccool commented:
We need to close the related issue as well prior to making the CR transition request.  I’ll try to do that, but if I can’t one of you should.
